content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -150,95 +145,95 @@ You should see a confirmation message from sensuctl.
 Created
 {{< /highlight >}}
 
-The `sensu-slack-handler` asset is now ready to use with Sensu.
-You can use sensuctl to see the complete asset definition.
+The `sensu-slack-handler` asset is now ready to use with Sensu.  Use sensuctl to see the complete asset definition.
 
 {{< highlight shell >}}
 sensuctl asset info sensu-slack-handler --format yaml
 {{< /highlight >}}
 
-_PRO TIP: You can use resources definition to create and update resources (like assets) using `sensuctl create --file filename.yaml`. See the [sensuctl docs][3] for more information._
+_PRO TIP: You can use resource definitions to create and update resources (like assets) using `sensuctl create --file filename.yaml`. See the [sensuctl docs][5] for more information._
 
 **3. Create a Sensu Slack handler**
 
 Open the `sensu-slack-handler.json` handler definition provided with the sandbox, and edit the definition to include your Slack channel, webhook URL, and the `sensu-slack-handler` asset.
 
 {{< highlight shell >}}
 "env_vars": [
-  "KEEPALIVE_SLACK_WEBHOOK=https://hooks.slack.com/services/AAA/BBB/CCC",
+  "KEEPALIVE_SLACK_WEBHOOK=https://hooks.slack.com/services/ABC",
```

Comment:
So I think the point of `AAA/BBB/CCC` was that Slack webhooks follow the form of `someString/someOtherString/andYetAnotherString`. For example: `https://hooks.slack.com/services/K04L63BU1/BTZ7FGFW/WwZjiHpaeDDiXmk3QZUzlrkO`. That said, I don't think it is too important, just an FYI. If a user has a webhook URL, they'll use the entire thing.

Reply:
No, I think you're correct. I will change it back.